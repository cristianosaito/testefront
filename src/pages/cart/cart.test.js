import React from 'react'
import { render } from '@testing-library/react'
import { Provider } from 'react-redux'
import configureStore from 'redux-mock-store'
import { BrowserRouter as Router } from 'react-router-dom'

import Cart from './index'

const mockStore = configureStore([])

const store = mockStore({})

test('Should match snapshot', () => {
  const container = render(
    <Provider store={store}>
      <Router>
        <Cart />
      </Router>
    </Provider>
  )
  expect(container).toMatchSnapshot()
})
