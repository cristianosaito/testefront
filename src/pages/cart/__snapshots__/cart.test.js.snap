// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container"
      >
        <h2>
          PRODUTOS
        </h2>
        <div>
          <ul
            class="info-table"
          >
            <li
              class="info"
            >
              <div
                class="card"
              >
                <p
                  class="info-item"
                >
                  PRODUTOS
                </p>
                <p
                  class="price"
                >
                  <span />
                </p>
              </div>
            </li>
            <li
              class="info"
            >
              <div
                class="card"
              >
                <p
                  class="info-item"
                >
                  FRETE
                </p>
                <p
                  class="price"
                >
                  <span />
                </p>
              </div>
            </li>
            <li
              class="info"
            >
              <div
                class="card"
              >
                <p
                  class="info-item-orange"
                >
                  DESCONTO
                </p>
                <p
                  class="price"
                >
                  <span />
                </p>
              </div>
            </li>
            <li
              class="info"
            >
              <div
                class="card"
              >
                <p
                  class="info-item-bold"
                >
                  TOTAL
                </p>
                <p
                  class="price"
                >
                  <span />
                </p>
              </div>
            </li>
          </ul>
        </div>
        <div>
          <a
            href="/payment"
          >
            <button
              class="sc-bdnxRM bVCTQC"
            >
              SEGUIR PARA O PAGAMENTO
            </button>
          </a>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container"
    >
      <h2>
        PRODUTOS
      </h2>
      <div>
        <ul
          class="info-table"
        >
          <li
            class="info"
          >
            <div
              class="card"
            >
              <p
                class="info-item"
              >
                PRODUTOS
              </p>
              <p
                class="price"
              >
                <span />
              </p>
            </div>
          </li>
          <li
            class="info"
          >
            <div
              class="card"
            >
              <p
                class="info-item"
              >
                FRETE
              </p>
              <p
                class="price"
              >
                <span />
              </p>
            </div>
          </li>
          <li
            class="info"
          >
            <div
              class="card"
            >
              <p
                class="info-item-orange"
              >
                DESCONTO
              </p>
              <p
                class="price"
              >
                <span />
              </p>
            </div>
          </li>
          <li
            class="info"
          >
            <div
              class="card"
            >
              <p
                class="info-item-bold"
              >
                TOTAL
              </p>
              <p
                class="price"
              >
                <span />
              </p>
            </div>
          </li>
        </ul>
      </div>
      <div>
        <a
          href="/payment"
        >
          <button
            class="sc-bdnxRM bVCTQC"
          >
            SEGUIR PARA O PAGAMENTO
          </button>
        </a>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
